tests/phpunit/CRM/Mailing/BAO/MailingTest.php

```diff
@@ -290,7 +290,7 @@ public function testgetRecipientsEmailGroupIncludeExclude() {
       }
       else {
         $groupIDs[$i] = $this->smartGroupCreate([
-          'formValues' => ['last_name' => (($i == 6) ? 'smart5' : 'smart' . $i)],
+          'form_values' => ['last_name' => (($i == 6) ? 'smart5' : 'smart' . $i)],
```

Comment:
I think we should add an api alias to the v3 api so that formValues still works  - since that's how we roll with v3 - it's just adding to the spec

$spec['form_values']['api.aliases'] = ['formValues'];

& the test would be that the above change would not be needed - acutally that's only true if the above calls apiv3 but still

Reply:
@eileenmcnaughton I've added the api alias although `formValues` was never officially supported by the api; the api expected `form_values`, so I've just pushed that expectation down to the BAO as well.

Reply:
cool